tools/BUCK.bzl

```diff
@@ -288,3 +288,20 @@ def define_tools_targets(
             ":autograd",
         ],
     )
+
+    python_test(
+        name = "test_torchgen_executorch",
+        srcs = [
+            "test/test_executorch_custom_ops.py",
+            "test/test_executorch_gen.py",
+            "test/test_executorch_signatures.py",
+            "test/test_executorch_types.py",
+            "test/test_executorch_unboxing.py",
+        ],
+        contacts = contacts,
+        visibility = ["PUBLIC"],
+        deps = [
+            torchgen_deps,
+            "fbsource//third-party/pypi/expecttest:expecttest",
```

Comment:
oops here you should do
```
third_party("expecttest"),
```
and also add expecttest to third_party definition.

Reply:
Yeah, it's also ok do a quick forward fix in this case as BUCK build only runs periodically.

Reply:
https://github.com/pytorch/pytorch/pull/96285 to remove first

Reply:
@huydhn @larryliu0820 Could you please review https://github.com/pytorch/pytorch/pull/96285 again?  I also added the third-party part.

Thank you!